### PR TITLE
Add icons for key features

### PR DESCRIPTION
**Pull request by ee7**
`configlet lint` now insists that the `icon` key is present for each key feature.

The list of available icons can be seen [here](https://github.com/exercism/docs/issues/189).

Feel free to change the icons to whatever is preferred.

---

This PR proposes:

`fast` Performance
Rust is blazingly fast and memory-efficient; it has no runtime or garbage collector.

`stable` Reliability
Rust’s rich type system and ownership model guarantee memory-safety and thread-safety.

`productive` Productivity
Rust has great documentation, a friendly compiler with useful error messages, and top-notch tooling.

`tooling` Cargo
Rust's package manager and build tool is best-in-class.

`cross-platform` Write Once, Run Anywhere
Rust compiles by default to a single static executable, and cross-compilation is easy.

`fun` ❤︎
Stack Overflow's most-loved language every year since 2016.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1850,26 +1850,32 @@
   ],
   "key_features": [
     {
+      "icon": "fast",
```
Comment (ee7) [inline config.json:1853]: `fast` `fast`

File: config.json [hunk 2]
```diff
@@ -1850,26 +1850,32 @@
...
       "title": "Performance",
       "content": "Rust is blazingly fast and memory-efficient; it has no runtime or garbage collector."
     },
     {
+      "icon": "stable",
```
Comment (ee7) [inline config.json:1858]: `stable` `stable`

You may prefer e.g.:

`immutable` `immutable`

File: config.json [hunk 3]
```diff
@@ -1850,26 +1850,32 @@
...
       "title": "Reliability",
       "content": "Rust’s rich type system and ownership model guarantee memory-safety and thread-safety."
     },
     {
+      "icon": "productive",
```
Comment (ee7) [inline config.json:1863]: `productive` `productive`

You may prefer e.g.:

`evolving` `evolving`

File: config.json [hunk 4]
```diff
@@ -1850,26 +1850,32 @@
...
       "title": "Productivity",
       "content": "Rust has great documentation, a friendly compiler with useful error messages, and top-notch tooling."
     },
     {
+      "icon": "tooling",
```
Comment (ee7) [inline config.json:1868]: `tooling` `tooling`

File: config.json [hunk 5]
```diff
@@ -1850,26 +1850,32 @@
...
       "title": "Cargo",
       "content": "Rust's package manager and build tool is best-in-class."
     },
     {
+      "icon": "cross-platform",
```
Comment (ee7) [inline config.json:1873]: `cross-platform` `cross-platform`

You may prefer e.g.

`portable` `portable`

File: config.json [hunk 6]
```diff
@@ -1850,26 +1850,32 @@
...
       "title": "Write Once, Run Anywhere",
       "content": "Rust compiles by default to a single static executable, and cross-compilation is easy."
     },
     {
+      "icon": "fun",
```
Comment (ee7) [inline config.json:1878]: `fun` `fun`

You may prefer e.g.:

`community` `community`

**Review by coriolinus — APPROVED** [review]
These look great, thanks!

**Pull request merged**